README.md

```diff
@@ -15,96 +15,48 @@ This project also uses [Node.js](https://nodejs.org), [Kettle](https://github.co
-    - a `ui` grade called *menu* for top-level links and controls, with an associated template called `menu.handlebars`.
-    - a component for UIO with some associated events to dynmically redraw the page contents when the language is changed, and
-    - a component for each `ui` in the tool and manages communication of relevant information between them.
-- `sjrk.storyTelling.page.storyBrowse` for the Browse page, has a `storyBrowser`
-- `sjrk.storyTelling.page.storyView` for the Browse page, has a `storyViewer`
-- `sjrk.storyTelling.page.storyBrowse` for the Edit page, has a `storyEditor` and a `storyViewer` which together form the story authoring environment. This file also contains some events, listeners and functions to handle story submission to the database
-- `sjrk.storyTelling.block.singleFileUploader` provides some wiring for the Edit page to upload files to the server
+* Run `node .\index.js` to launch the server
 
 ### Theme Customization
-The "base" JavaScript files included in this repository, located in the `ui/src/js` directory, are intended to provide a bare-bones implementation of the Storytelling Tool. The interface is extensible and customizable to enable the creation of new experiences, themes and story contexts with the tool. To create a custom theme, follow these steps:
-- Create a new folder for code and assets associated with the new theme (for examples, please see the `ui/src/karisma` or `ui/src/learningReflections` folders)
-- Create a JavaScript file which contains extensions of the page grades (e.g. for `sjrk.storyTelling.page.storyEdit`, add a new grade `sjrk.storyTelling.cuteCats.storyEdit` which has the former as one of its gradeNames). These extensions could include new `ui` components for new sections of the page, new events or other functionality. The name of the JavaScript file must match the folder name, e.g. `ui/src/cuteCats/js/cuteCats.js`
-- Add any new associated handlebars templates to be used by new UI components
-- If your new content contains any new wording, you can create new message bundle files or add them to the existing message files (in the `ui/src/messages` directory) and refer to those message names in the handlebars template(s). Please note that a `ui` component specifies and loads only one message bundle at a time.
+The "base" files included in this repository, located in the `themes/base` directory, are intended to provide a bare-bones implementation of the Storytelling Tool. The interface is extensible and customizable to enable the creation of new experiences, themes and story contexts with the tool. To create a custom theme, follow these steps:
+- Set the `theme` value in the server config file (see [Running the site](#Running-the-site)) to the name of your new theme. For the sake of this example, we'll use `cuteCats`.
+- Create a new folder for code and assets associated with the new theme (for examples, please see the `themes/karisma` or `themes/learningReflections` folders)
```

Comment:
Please also describe the name of the new folder needs to be same as the `theme` value defined in the server config file.